components/messaging/CurrentConversation.tsx

```diff
@@ -3,25 +3,11 @@
 import { MessageType } from '@/utils/messaging/messagingTypes';
 import MessageCard from './MessageCard';
 import MessageForm from './MessageForm';
-import { useContext, useEffect } from 'react';
+import { useContext } from 'react';
 import useConversation from '../../app/(dashboard)/conversations/useConversation';
 
 const CurrentConversation: React.FC = () => {
-  const { allConversations, currentConversation, setCurrentConversation } =
-    useContext(useConversation);
-
-  useEffect(() => {
-    const highestId = Math.max(
-      ...allConversations.map((conversation) => conversation.conversation_id)
-    );
-
-    setCurrentConversation &&
-      setCurrentConversation(
-        allConversations?.filter(
-          (conversation) => conversation.conversation_id === highestId
-        )[0]
-      );
-  }, []);
+  const { currentConversation } = useContext(useConversation);
 
```

Comment:
We should talk about this, but I think it's important that we keep this (it's also been refactored and is up to date now in the dev branch. This sets the open conversation when the component is first mounted and is different then the eventhandler which updates the current open conversation on click.